client/src/components/user/dashboard.js

```diff
@@ -198,15 +204,15 @@ class Dashboard extends React.Component {
         </div>;
     return (
       <div className="h-100 d-flex flex-column">
-        <header className="navbar p-2 d-flex justify-content-between align-items-center text-dark border-bottom shadow-sm">
+        <header className="navbar py-2 px-4 d-flex justify-content-between align-items-center text-dark border-bottom shadow-sm">
           <a className="navbar-brand d-flex text-dark" href="#changeThis">
             <img src={logo} alt="wateringCanIcon" height="60" />
             <div className="align-self-center d-none d-md-block">Water Your Subs</div>
           </a>
-          <h1 className="display-4 fw-normal">{`${new Date().toDateString()}`}</h1>
+          <h1 className="h3 fw-normal">{concatenatedString}</h1>
```

Comment:
this makes the header always smaller, i was suggesting to only make it smaller when in mobile view